src/Utils.agda

```diff
@@ -3,32 +3,20 @@
 module Utils where
```

Comment:
Maybe this module should be removed since it's been split up? Or just become a list of re-exports of all the `Utils.*` modules?